pipelines/build-testapp.yml

```diff
@@ -94,9 +94,256 @@ stages:
+            }
+          }
+
+    # - task: EsrpCodeSigning@3
+    #   inputs:
+    #     ConnectedServiceName: 'ml-inference ESRP CodeSigning service connection'
+    #     FolderPath: 'test_apps/android/test_app/app/build/outputs/apk/release/'
+    #     Pattern: '*-release.apk'
+    #     UseMinimatch: true
+    #     signConfigType: 'inlineSignParams'
+    #     inlineOperation: |
+    #       [
+    #       {
+    #           "KeyCode" : "CP-232601-Java",
+    #           "OperationCode" : "AndroidSign",
+    #           "Parameters" : {},
+    #           "ToolName" : "sign",
+    #           "ToolVersion" : "1.0"
+    #       },
+    #       {
+    #           "KeyCode" : "CP-232601-Java",
+    #           "OperationCode" : "JavaVerify",
+    #           "Parameters" : {},
+    #           "ToolName" : "sign",
+    #           "ToolVersion" : "1.0"
+    #       }
+    #       ]
+    #     SessionTimeout: '60'
+    #     MaxConcurrency: '50'
+    #     MaxRetryAttempts: '5'
+
+    - task: PowerShell@2
+      displayName: 'Copy '
+      inputs:
+        pwsh: true
+        targetType: 'inline'
+        failOnStderr: false
+        script: |
+          cd $(Build.SourcesDirectory)/test_apps/android/test_app
+          $abis = "x86", "x86_64", "arm64-v8a", "armeabi-v7a"
+          foreach ($abi in $abis) {
+            $artifactName="ort_1_14_0_android_$abi"
+            $outputFolder="$(Build.ArtifactStagingDirectory)/$artifactName"
+            mkdir -Force $outputFolder
+            cp app/build/outputs/apk/release/app-$abi-release.apk $outputFolder
+          }
+
+    - task: PowerShell@2
+      displayName: 'validate test apps for Android'
+      inputs:
+        pwsh: true
+        targetType: 'inline'
+        failOnStderr: false
+        script: |
+          cd $(Build.SourcesDirectory)/python
+          python -m pip install -e .
+          $env:APPCENTER_TOKEN='$(APPCENTER-TOKEN-UNIVERSAL)'
+          $env:IOS_APPCENTER_OWNER='$(APPCENTER-OWNER)'
+          $env:IOS_APPCENTER_APP='$(IOS-APPCENTER-APP)'
+          $env:ANDROID_APPCENTER_OWNER='$(APPCENTER-OWNER)'
+          $env:ANDROID_APPCENTER_APP='$(ANDROID-APPCENTER-APP)'
+          npm install -g appcenter-cli 2>&1
+          python $(Build.SourcesDirectory)/python/test/mobile/test_mobile.py
+          if ($LASTEXITCODE -ne 0) {
+            echo "Android validation failed"
+            exit 1
+          }
+
+    - publish: $(Build.ArtifactStagingDirectory)/ort_1_14_0_android_x86
+      artifact: ort_1_14_0_android_x86
+    - publish: $(Build.ArtifactStagingDirectory)/ort_1_14_0_android_x86_64
+      artifact: ort_1_14_0_android_x86_64
+    - publish: $(Build.ArtifactStagingDirectory)/ort_1_14_0_android_arm64-v8a
+      artifact: ort_1_14_0_android_arm64-v8a
+    - publish: $(Build.ArtifactStagingDirectory)/ort_1_14_0_android_armeabi-v7a
+      artifact: ort_1_14_0_android_armeabi-v7a
+
+- stage: BuildTestAppOnMacOS
+  dependsOn: 
+  jobs:
+  - job: test_app_ios
```

Comment:
Also please disable this job before signing task is ready because we will run the task and release artifacts regularly.

Reply:
disabled the publish task, I think the pipeline should still be run to ensure the android/ios packages are correct